modules/core/src/lib/deck-renderer.js

```diff
@@ -0,0 +1,69 @@
+import log from '../utils/log';
+import LayersPass from '../passes/layers-pass';
+
+const LOG_PRIORITY_DRAW = 2;
+
+export default class DeckRenderer {
```

Comment:
Total Nit, should be settled in the API audit of course, but personally I would prefer this classed to be named `DrawManager` or maybe `RenderManager`.

The reason is mostly that it fits with our current `...Manager` naming scheme, and that `...Renderer` gives me connotations of a class that does things like support rendering towards different backends (e.g. WebGLRenderer, CanvasRenderer, SVGRenderer) or a class that manages a lot of rendering resources.

Reply:
Welcome for any new name but it is not a manager, it doesn't manage resources or data, more like an action executor, get resources or data from managers and do rendering job.